codespell_lib/data/dictionary.txt

```diff
@@ -31604,8 +31604,10 @@ transprts->transports
 transpsition->transposition
 transsend->transcend
 transtion->transition
+transtioned->transitioned
 transtions->transitions
 transtition->transition
+transtition->transitioned
```

Comment:
This word `transtition` is already in the list right above it. What you are proposing is  `transtition->transition, transitioned,` ?

Reply:
Ah, I meant `transtitioned` → `transitioned`. I've fixed it in [`9c83565`](https://github.com/codespell-project/codespell/pull/2288/commits/9c835657f104f08b6b71ccdcf826292fff769ad0).